### PR TITLE
blog: kyverno playground

### PR DESCRIPTION
**Pull request by eddycharly**
## Related issue #

kyverno playground blog post 🤷

### CODE REVIEW

**Comment by eddycharly** [comment]
cc @fjogeleit

**Comment by eddycharly** [comment]
TODO: mention this old issue https://github.com/kyverno/kyverno/issues/2378

File: content/en/blog/general/playground/the-missing-piece/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+I think the Playground is one of the most wanted and oldest feature request.
+
+Since the beginning of Kyverno, we were regularly asked if we had plans to create an interactive website to play with Kyverno directly from a browser. Users wanted to experiment without the burden of spinning up a cluster, installing the CLi and write some yaml files, dig out the [policies catalog](https://kyverno.io/policies) to find policies to test, etc...
+
+While the feature request was legitimate, we never found the time to think about it more than a few minutes and the challenge to create such a project seemed pretty high, there was always more high priority things to do.
+
+This was until we came back from KubeCon EU in Amsterdam !
+
+It was my first KubeCon and i had the chance to meet in real life with part of the Kyverno team, Jim Bugwadia (, )creator of Kyverno and co-founder of Nirmata, incidentally he is also my boss), and Frank (long time Kyverno maintainer and creator or the policy reporter and other projects, he's the man behind falco sidekick for example, just that).
```
Comment (fjogeleit) [inline content/en/blog/general/playground/the-missing-piece/index.md:22]: Falcosidekick UI together with Thomas, Falcosidekick itself was created by Thomas (@issif) ❤️

**Comment by chipzoller** [comment]
Keeping or ?

**Comment by eddycharly** [comment]
> Keeping or ?

As discussed on slack, i will post it on my personal blog as the tone is not appropriate for the official kyverno blog.

**Comment by chipzoller** [comment]
Ok

**Pull request closed without merging**